packages/edge-login-ui-rn/src/reducers/WorkflowReducer.ts

```diff
@@ -1,62 +1,114 @@
-import { WorkflowName, workflows } from '../constants/workflows'
+import { SceneNames } from '../constants/Scenes'
 import { Action } from '../types/ReduxTypes'
 
-export interface WorkflowState {
-  readonly currentKey: WorkflowName
-  readonly currentSceneIndex: number
+export interface SceneState {
+  readonly currentScene: SceneNames
 }
 
-const initialState: WorkflowState = {
-  currentKey: 'loadingWF',
-  currentSceneIndex: 0
+const initialState: SceneState = {
+  currentScene: 'LoadingScene'
 }
 
-export function workflow(
-  state: WorkflowState = initialState,
+export function scene(
+  state: SceneState = initialState,
   action: Action
-): WorkflowState {
+): SceneState {
   switch (action.type) {
-    // Generic workflow actions:
-    case 'WORKFLOW_BACK': {
-      let { currentSceneIndex } = state
-      if (currentSceneIndex > 0) {
-        --currentSceneIndex
-      }
-      return { ...state, currentSceneIndex }
-    }
-    case 'WORKFLOW_NEXT': {
-      let { currentSceneIndex } = state
-      if (currentSceneIndex + 1 < workflows[state.currentKey].length) {
-        ++currentSceneIndex
-      }
-      return { ...state, currentSceneIndex }
-    }
-
-    // Actions for launching scenes:
     case 'OTP_ERROR':
-      return { ...state, currentKey: 'otpWF', currentSceneIndex: 0 }
+      return {
+        ...state,
```

Comment:
There are a bunch of `...state` entries in this file, but they don't do anything now, because there are no other properties besides `currentScene` that we need to preserve.